### PR TITLE
Modem: Add GPS time and update Overview

### PR DESCRIPTION
**Pull request by albsod**
- Group Mobile network documentation under one heading
- Add Global Positioning System (GPS) with Time as sub-heading
- Explain that the modem typically includes both mobile and gps functionality

### CODE REVIEW

File: docs/interfaces/modem/modem.md [hunk 1]
```diff
@@ -96,5 +98,14 @@ error:
 +CME ERROR: SIM not inserted
 ```
 
+## Global Positioning System (GPS)
+
+### Time
+
+The GPS time can be extracted from the `$GPRMC` string outputted by the GPS.
```
Comment (RikardO-HM) [inline docs/interfaces/modem/modem.md:105]: maybe add that gps-parser is possible to use in most of our system or can be download using opkg.

Comment (albsod) [reply docs/interfaces/modem/modem.md:105]: Do you mean [gpsd](https://gpsd.io/)?

Comment (RikardO-HM) [reply docs/interfaces/modem/modem.md:105]: no but looks like we have gpsd on HMX but not gps-parser (i tried to find it in opkg list and maybe it is an older tool). 
gps2udp       gpscat        gpscsv        gpsctl        gpsd          gpsdctl       gpsdebuginfo  gpsdecode     gpsfake       gpsmon        gpspipe       gpsplot       gpsprof       gpsrinex      gpssnmp       gpssubframe

Comment (RikardO-HM) [reply docs/interfaces/modem/modem.md:105]: it is okay

**Review by RikardO-HM — APPROVED** [review]

**Pull request merged**